packages/material-ui/src/Menu/Menu.js

```diff
@@ -8,7 +8,7 @@ import Popover from '../Popover';
 import MenuList from '../MenuList';
 import warning from 'warning';
 import ReactDOM from 'react-dom';
-import { setRef } from '@material-ui/core/utils/reactHelpers';
```

Comment:
😱 Murphy!

Reply:
This caused 
> @material-ui/core: parsed: -0.15% heart_eyes, gzip: -0.15% heart_eyes

Reply:
Need to investigate why the script didn't fail. Preferably rollup would exit with non-zero.

Reply:
During the last release, I have seen unusual rollup warnings.

Reply:
I can see the same warning in the next build CI output:
![Capture d’écran 2019-04-25 à 22 56 54](https://user-images.githubusercontent.com/3165635/56768008-774c4100-67ad-11e9-86e1-c28a4b62fdc9.png)
The warnings are gone on this branch 👌. 
@TrySound Do you know if we can make these warnings failing?

Reply:
Yes, you can
```js
onwarn(warning, warn) {
  if (condition1) {
    return;
  }
  if (condition2) {
    return warn(warning)
  }
  throw Error(warning.message);
},
```

Reply:
Awsome, I'm trying this out :)

Reply:
![Capture d’écran 2019-04-26 à 21 15 11](https://user-images.githubusercontent.com/3165635/56831048-69a7c180-6868-11e9-874c-dd70fa40b12f.png)